RecoTracker/MkFitCMS/src/MkStdSeqs.cc

```diff
@@ -137,6 +137,12 @@ namespace mkfit {
       std::vector<float> d0(ns);
       int i1, i2;  //for the sorting
 
+      axis_pow2_u1<float, unsigned short, 16, 8> ax_phi(-Const::PI, Const::PI);
+      axis<float, unsigned short, 8, 8> ax_eta(-3.0, 3.0, 30u);
+      binnor<unsigned int, decltype(ax_phi), decltype(ax_eta), 24, 8> b(ax_phi, ax_eta);
```

Comment:
single-letter local variables should be avoided (usually OK for small scopes of 10-20 lines, but here it's not the case.